src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -1,27 +1,57 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Implementation.CodeCleanup;
+using Microsoft.CodeAnalysis.Editor.Shared.Options;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Extensions;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 using VSCommanding = Microsoft.VisualStudio.Commanding;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Formatting
 {
-    internal partial class FormatCommandHandler
+    internal partial class FormatCommandHandler : ForegroundThreadAffinitizedObject
     {
+        private bool _infoBarOpen = false;
         public VSCommanding.CommandState GetCommandState(FormatDocumentCommandArgs args)
         {
             return GetCommandState(args.SubjectBuffer);
         }
 
-        public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionContext context)
+        private void ShowGoldBarForCodeCleanupConfiguration(Workspace workspace)
         {
-            return TryExecuteCommand(args, context);
+            AssertIsForeground();
+
+            // If the gold bar is already open, do not show
+            if (_infoBarOpen)
+            {
+                return;
+            }
+
+            _infoBarOpen = true;
+
+            var optionPageService = workspace.Services.GetRequiredService<IOptionPageService>();
+            var infoBarService = workspace.Services.GetRequiredService<IInfoBarService>();
+            infoBarService.ShowInfoBarInGlobalView(
+                EditorFeaturesResources.Code_cleanup_is_not_configured,
+                new InfoBarUI(EditorFeaturesResources.Configure_it_now,
+                              kind: InfoBarUI.UIKind.Button,
+                               () =>
+                               {
+                                   optionPageService.ShowFormattingOptionPage();
+                                   _infoBarOpen = false;
+                               }),
```

Comment:
indentation of the lambda does not match the other arguments.

Reply:
fixed